app/pages/events/event_view.php

```diff
@@ -95,8 +95,11 @@
         </div>
 
         <?php if ($event->bulletin_url): ?>
-            <a href="<?= $event->bulletin_url ?>" target="_blank"> <i class="fa fa-paperclip"></i> Bulletin
-                <i class="fa fa-external-link"></i></a>
+
+            <p>
```

Comment:
`div` will also turn the element into a block so it will stack vertically, but without the extra spacing. Maybe you want to use that instead?

Reply:
I leave it with the `<p>`